app/models/explainer.rb

```diff
@@ -107,7 +107,7 @@ def self.search_by_similarity(text, language, team_id, limit)
     models_thresholds = Explainer.get_alegre_models_and_thresholds(team_id)
     context = {
       type: 'explainer',
-      team: Team.find(team_id).slug
+      team: Team.find(team_id).id
```

Comment:
Same here - team_id instead of team I think.

Reply:
Yes, no need to load the team object from the ID and to then get the ID... just replace this whole thing by the `team_id` variable.